.github/workflows/minimal.yml

```diff
@@ -24,6 +24,7 @@ jobs:
       shell: "bash -l {0}"
       env:
         ZARR_V3_EXPERIMENTAL_API: 1
+        ZARR_V3_SHARDING: 1
```

Comment:
Sorry, review got caught up on trying to figure out whether or not a second environment variable was needed, and then whether or not we need some more general mechanism, etc. etc. I assume it's not an issue to introduce a new one, but moving forward, we should probably make this easier on ZEP developers.

Reply:
I felt that this was quite an easy change, but true, this could be simplified in the future (e.g. a single variable for all enabled ZEPs, and a special value to enable all, which is used in CI).